test/integration/integration/constants/constants.go

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package constants
+
+// gateway service properties
+const (
+	GatewayServiceName      string = "apk-test-wso2-apk-router-service"
+	GatewayServiceNamespace string = "apk"
+	GatewayServicePort      int16  = 9090
```

Comment:
is it intentional? to use the unsecured gateway port?

Reply:
Only used http port for this initial PR. Only tried to get a single test running. Will add https port (9095) in next PR that include test cases